final_prep/README.md

```diff
@@ -44,31 +44,31 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1097" alt="Screen Shot 2022-04-20 at 7 55 55 PM" src="https://user-images.githubusercontent.com/80866068/164814995-1da85819-2a28-47b0-ad7d-8d9e6eb838f2.png">
+- <img width="1140" alt="Screen Shot 2022-04-20 at 7 56 08 PM" src="https://user-images.githubusercontent.com/80866068/164815067-f52448f4-111a-417e-becc-5f2f1babd5e8.png">
+- <img width="1144" alt="Screen Shot 2022-04-20 at 7 56 19 PM" src="https://user-images.githubusercontent.com/80866068/164815104-3da3d308-1d76-4fe9-bb2c-353674a30ca7.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Click me!](https://gist.github.com/jaredhardinger/bc2cc07e11c50591e7c50e64d3ff024f)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: [Click me!](https://gist.github.com/jaredhardinger/5a735c6824515ddbcc04c2099fe70682)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: [Click me!](https://gist.github.com/jaredhardinger/5d8f8f00526e0db1d79810f39fc54edd)
```

Comment:
Ok I saw spending time with partner on there - is that something you need to carve out specific times for on your calendar so you hold yourself to doing that, and don't get in trouble with them? Weekly event, or daily smaller event? Looks different for everyone but just a suggestion:) I'll bet they will be understanding since been through Turing, but you still want to take care of the relationship AND it's probably important for you to get that together time, to help keep the momentum up through hard weeks at Turing!!